src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -1551,10 +1550,11 @@ public String getUrl() {
          */
         @SuppressWarnings("unused") // used from jelly pages
         public Node getBuiltOn() {
-            if (builtOn == null || builtOn.equals("")) {
-                return Jenkins.getInstance();
+            Jenkins j = Jenkins.getInstance();
+            if (builtOn == null || builtOn.isEmpty() || j == null) {
+                return j;
```

Comment:
In fact it could have returned null to begin with (from `getNode`), but I will mark it for clarity.